boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -21,32 +21,7 @@ has an USB-C port for programming and debugging, integrated battery charging
 and an on-board antenna. The fitted U.FL external antenna connector can be
 enabled by moving a 0-ohm resistor.
 
-Connections and IOs
-===================
-
-The ``ttgo_t8c3`` board target supports the following hardware features:
-
-+-----------+------------+------------------+
-| Interface | Controller | Driver/Component |
-+===========+============+==================+
-| PMP       | on-chip    | arch/riscv       |
-+-----------+------------+------------------+
-| INTMTRX   | on-chip    | intc_esp32c3     |
-+-----------+------------+------------------+
-| PINMUX    | on-chip    | pinctrl_esp32    |
-+-----------+------------+------------------+
-| USB UART  | on-chip    | serial_esp32_usb |
-+-----------+------------+------------------+
-| GPIO      | on-chip    | gpio_esp32       |
-+-----------+------------+------------------+
-| UART      | on-chip    | uart_esp32       |
-+-----------+------------+------------------+
-| I2C       | on-chip    | i2c_esp32        |
-+-----------+------------+------------------+
-| SPI       | on-chip    | spi_esp32_spim   |
-+-----------+------------+------------------+
-| TWAI      | on-chip    | can_esp32_twai   |
-+-----------+------------+------------------+
+.. zephyr:board-supported-hw::
```

Comment:
possibly align with the structure from other ESP32 (and just most boards, really) boards -- i.e rename Functional Description heading to "Hardware", and then do this?

```suggestion
Supported Features
==================

.. zephyr:board-supported-hw::
```